java-practice-methods/.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,5 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0" is_locked="false">
+    <option name="myName" value="Project Default" />
+  </profile>
+</component>
```

Comment:
Unnecessary file (this is the project settings file for IntelliJ IDEA).
Fix it:
- create file .gitignore
- add to the file .idea/